### DISCUSSION TITLE
Oculus GO controller support!!

### DISCUSSION THREAD

**Discussion by ghost** [alvr-org/ALVR#579] [General]
Kindly please implement Oculus Go controller support like the original ALVR. 

https://github.com/polygraphene/ALVR

Thank You.

**Comment by zmerp** [comment]
Unfortunately this has low priority right now. Other people can upvote this suggestion so I can gauge the interest.

**Comment by Thanderbirth** [comment]
very need

**Reply by ontwikkelfabriek** [reply]
@zarik5 

> Unfortunately this has low priority right now. Other people can upvote this suggestion so I can gauge the interest.

So if the Go controller is not supported then how can I use my Oculus Go with ALVR?

**Comment by ghost** [comment]
Mouse/keyboard then gamepad.

**Comment by ontwikkelfabriek** [comment]
+1
